src/ndarray/ndarray_function.cc

```diff
@@ -32,19 +32,34 @@
 
 namespace mxnet {
 namespace ndarray {
+template<typename DType>
+void OMPCopy(const TBlob &from, TBlob *to, const index_t size) {
+  DType* dst_dptr = to->dptr<DType>();
+  DType* src_dptr = from.dptr<DType>();
+  #pragma omp parallel for num_threads(engine::OpenMP::Get()->GetRecommendedOMPThreadCount())
+  for (index_t i = 0; i < size; ++i) {
+    dst_dptr[i] = src_dptr[i];
+  }
+}
+
 template<>
 void Copy<cpu, cpu>(const TBlob &from, TBlob *to,
                     Context from_ctx, Context to_ctx,
                     RunContext ctx) {
   MSHADOW_TYPE_SWITCH(to->type_flag_, DType, {
     if (to->type_flag_ == from.type_flag_) {
-        mshadow::Copy(to->FlatTo1D<cpu, DType>(),
-                      from.FlatTo1D<cpu, DType>());
+      index_t copy_block_size = dmlc::GetEnv("MXNET_CPU_PARALLEL_COPY_SIZE", 200000);
```

Comment:
static?

Reply:
there is a API:` mx.test_utils.set_env_var`  to set env var. it may can't work repeatly if add static modifier. like my [test case] (https://github.com/XiaotaoChen/incubator-mxnet/blob/cxt-test/tests/cxt-test/parallel-copy/test_ndarray_copy.py#L89-L92). what's your opinion？ @szha

Reply:
It seems wasteful to get the variable every time it launches a copy as people don't usually change this setting at runtime. Try testing it with some other approach.

Reply:
ok, I'll do it.